### ISSUE TITLE
Hangs on startup (displaying blue screen) if DF is not running

### ISSUE THREAD

**Issue by lethosor** [JapaMala/armok-vision#1]
May be wine-specific

**Comment by JapaMala** [comment]
Yeah, it attempts once to connect on startup, and doesn't try again. Need to change that.

**Comment by PeridexisErrant** [comment]
As of v0.11.3, if started without DF Armok Vision simply shows the nighttime skybox.  Ideally it would show a message "DF not found.  Is DF running, with DFHack and remotefortressreader installed?" and offer a button "press to retry connection".

**Comment by JapaMala** [comment]
Agreed. I'll work on this.
